docs/design.adoc

```diff
@@ -13,6 +13,15 @@ covered in v2 documentation.
 
 == Components
 
+=== Collateral token
+
+ERC-20 token deposited in the pool by underwriters and accepted by the pool as
```

Comment:
I would change the order, ie. first mention that it is accepted by the pool and then these tokens can be accepted as a deposit. How about something like that:
"ERC-20 token which is accepted by the pool as collateral and used as a deposit token by underwriters."

Reply:
Yeah, makes sense! I'll update.